stylus/ui-base/fonts.styl

```diff
@@ -20,6 +20,10 @@ $font-title
 $font-labor
     font-family  Lato, sans-serif
 
+$font-smoothing
+    -moz-osx-font-smoothing grayscale
```

Comment:
I did the same as for `$font-labor` trying to keep the code coherent :) I can write directly it on the `body`